src/core/math/vec2/canonicalize.test.js

```diff
@@ -0,0 +1,20 @@
+const canonicalize = require('./canonicalize')
+const equals = require('./equals')
+const fromValues = require('./fromValues')
+const test = require('ava')
+
+const piVec = fromValues(3.141592653589793, 3.141592653589793)
+
+// This test is intended to be illustrative, and establish ground truth
+// It will need to be updated if ../../constants.spatialResolution changes.
+test('vec2: Canonicalizion quantizes to spatial resolution', t => {
```

Comment:
test is great except for spelling :) 
- canonicalization instead of Canonicalizion
- idempotent instead of idemponent

Reply:
Thanks for spotting those.

I have a little dyslexia, which makes it hard to see those in long words. :)

Reply:
no worries, I had to double check as well :)